next.config.js

```diff
@@ -8,5 +8,6 @@ module.exports = withCSS({
   publicRuntimeConfig: {
     AUTHENTICATION_SERVER_BASE_URL: process.env.AUTHENTICATION_SERVER_BASE_URL,
     RESOURCES_SERVER_BASE_URL: process.env.RESOURCES_SERVER_BASE_URL
-  }
+  },
+  webpack5: false
```

Comment:
This needs deleting: https://nextjs.org/docs/basic-features/built-in-css-support

zeit/next-css has been deprecated; NextJs recommends using built-in stylesheets implementation instead, then run on the default webpack 5. Next version of NextJs will remove support for webpack 4.

Reply:
done